config/genesis.go

```diff
@@ -0,0 +1,62 @@
+package config
+
+import (
+	"encoding/json"
+	"os"
+	"time"
+)
+
+// GenesisDoc representa el documento génesis con métodos para acceder a sus campos
+type GenesisDoc interface {
```

Comment:
we should put this in types, do we need an interface here? it should be specific to rollkit,